Heartcore release notes for April 2025 (#7022)
umbraco/UmbracoDocs: umbraco-heartcore/release-notes/2025-04-releasenotes.md

# April 2025
The following changes have been released to all Heartcore sites.

## Umbraco Workflow now available on Heartcore

> Umbraco Workflow is now included as part of the offering on Umbraco Heartcore!

Reviewer: ```suggestion
Umbraco Workflow is now included as part of the offering on Umbraco Heartcore.
```

This integration gives Heartcore users powerful content approval and publishing workflows right out of the box.
It makes it even easier to manage content governance in headless SaaS projects.

> What's Included?

Reviewer: ```suggestion
## What's Included?
```

> The free version of Umbraco Workflow is now automatically available to all Heartcore projects, enabling structured review and publishing flows.
> For more advanced needs, a licensed version of Umbraco Workflow is also available.
> It includes extra features and customization options and can be acquired it by contacting our sales team.

Reviewer: ```suggestion
The free version of Umbraco Workflow is now automatically available for all Heartcore projects, allowing for structured review and publishing processes. If you have more advanced needs, a licensed version of Umbraco Workflow is also available. This version includes additional features and customization options, which can be obtained by contacting the Sales Team.
```

> [Learn more about Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/)

Reviewer: ```suggestion
Learn more about [Umbraco Workflow](https://umbraco.com/products/add-ons/workflow/) and explore the comprehensive [Documentation](https://docs.umbraco.com/umbraco-workflow) to unlock its full potential.
```
